hawc/apps/riskofbias/api.py

```diff
@@ -67,12 +72,17 @@ def full_export(self, request, pk):
         Get all risk of bias/study evaluations for an assessment, including individual reviews.
         """
         self.get_object()
+        ser = ExportQuerySerializer(data=request.query_params)
+        ser.is_valid(raise_exception=True)
+        published_only = get_published_only(self.assessment, request)
         rob_name = self.assessment.get_rob_name_display().lower()
         exporter = exports.RiskOfBiasCompleteFlat(
```

Comment:
The published_only variable should be passed in here and handled in the class init.  It should be available in the kwargs dictionary on the instance.